CovidWatch iOS/ViewControllers/HowItWorks.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  HowItWorks.swift
+//  CovidWatch iOS
+//
+//  Created by Nikhil Kumar on 4/23/20.
+//  Copyright © 2020 IZE. All rights reserved.
+//
+
+import UIKit
+
+class HowItWorks: UIViewController {
+    @IBOutlet weak var page1: UIView!
+    @IBOutlet weak var page2: UIView!
+    @IBOutlet weak var page3: UIView!
+    @IBOutlet weak var page4: UIView!
+    @IBOutlet weak var howItWorksLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var setupButton: UIButton!
+    @IBOutlet var setupButtonHeight: NSLayoutConstraint!
+
+    override func updateViewConstraints() {
+        if let setupButtonHeight = self.setupButtonHeight {
+            setupButtonHeight.constant = (58.0/321.0) * contentMaxWidth
+        }
+        super.updateViewConstraints()
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // accessibility identifiers
+        setupAccessibilityAndLocalization()
+
+        UserDefaults.standard.set(true, forKey: Splash.onboardingStartedKey)
+        // since screen 4 does not have pages
+        if let page1 = self.page1,
+            let page2 = self.page2,
+            let page3 = self.page3,
+            let page4 = self.page4 {
+            page1.layer.cornerRadius = 10
+            page2.layer.cornerRadius = 10
+            page3.layer.cornerRadius = 10
+            page4.layer.cornerRadius = 10
+        }
+        if let setupButton = self.setupButton {
+            setupButton.layer.cornerRadius = 10
+            var buttonFontSize: CGFloat = 18
+            if screenHeight <= 568 {
+                buttonFontSize = 14
+            } else if screenHeight <= 667 {
+                buttonFontSize = 16
+            }
```

Comment:
Have you done an app supporting dynamic type before?  Im not very knowledgable on it, but I wonder if this stuff is going to make it harder to support dynamic type later.

Reply:
Apple has their own "Automatically Adjust Font Size" property, but I noticed it doesn't work with custom fonts (just Apple being Apple). Our best option is to extend `UIFontDescriptor` to manually set custom font sizes based on user's accessibility font size setting and screen width. For now though, I would say this is beyond the scope of this PR.